src/main/java/uk/gov/hmcts/reform/cpo/service/mapper/CasePaymentOrderMapper.java

```diff
@@ -4,10 +4,15 @@
 import uk.gov.hmcts.reform.cpo.data.CasePaymentOrderEntity;
 import uk.gov.hmcts.reform.cpo.domain.CasePaymentOrder;
 
+import java.util.List;
+
 @Mapper(componentModel = "spring")
 public interface CasePaymentOrderMapper {
 
     CasePaymentOrderEntity toEntity(CasePaymentOrder casePaymentOrder);
 
     CasePaymentOrder toDomainModel(CasePaymentOrderEntity casePaymentOrderEntity);
+
+    List<CasePaymentOrder> map(List<CasePaymentOrderEntity> casePaymentOrderEntities);
```

Comment:
let's follow naming conventions consistent with existing methods
```suggestion
    List<CasePaymentOrder> toDomainModelList(List<CasePaymentOrderEntity> casePaymentOrderEntities);
```

Reply:
DONE